packages/ui/src/components/ui-container.tsx

```diff
@@ -3,7 +3,7 @@ import { cn } from '../lib/utils.ts'
 
 export function UiContainer({ children, className, ...props }: ComponentProps<'div'>) {
   return (
-    <div className={cn('container mx-auto px-2 md:px-4 sm:px-6 lg:px-8', className)} {...props}>
+    <div className={cn('container mx-auto px-2 md:px-4 sm:px-6 lg:px-8 sm:max-w-xl', className)} {...props}>
```

Comment:
This change should not go here.

We don't want to apply this max-width across our app. We want this on onboarding so it should go in `onboading-ui-layout`.

No changes needed to `ui-container` in this PR or any of the others in #400. All tweaks should be local to the features.

Reply:
No need. 

Please just follow my request, setting the classname on `UiContainer`.

```tsx
    <div className="min-h-full w-full flex items-center justify-center">
      <UiContainer className="sm:max-w-xl">
        <Outlet />
      </UiContainer>
    </div>
```